pkg/ebpf/net_events.go

```diff
@@ -50,12 +51,12 @@ func (t *Tracee) processNetEvents(ctx gocontext.Context) {
 			}
 
 			if evtMeta.NetEventId == NetPacket {
-				captureData, err := parseCaptureData(payloadBytes)
+				captureData, err := parseCaptureData(payloadBytes, t.config.Debug)
```

Comment:
We shouldn't pass t.config.Debug to this helper, as it has nothing to do with the capture data bytes which are always the same